Index: app/src/main/java/com/example/mg/tryappkillan/form/StartActivity.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/java/com/example/mg/tryappkillan/form/StartActivity.java	(revision )
+++ app/src/main/java/com/example/mg/tryappkillan/form/StartActivity.java	(revision )
@@ -48,7 +48,9 @@
 import java.util.Date;
 import java.util.List;
 
-
+/**
+ * Created by mg on 28/03/17.
+ */
 
 
 public class StartActivity extends AppCompatActivity implements DataFragment.OnFragmentInteractionListener, GoogleApiClient.ConnectionCallbacks, GoogleApiClient.OnConnectionFailedListener {
